### PR TITLE
Makes sidebar resizable when at the guides tab

### PR DESCRIPTION
**Pull request by noahmay**
Adds the option to resize sidebar when the guides tab is open. Also replaces the library used for resizing (previously used with the node inspector). The new library has better performance since it directly modifies the DOM and then updates the state.

### CODE REVIEW

File: src/browser/components/TabNavigation/Navigation.tsx [hunk 1]
```diff
@@ -172,10 +175,11 @@ class Navigation extends Component<NavigationProps, NavigationState> {
 
     const drawerIsVisible = this.state.transitionState !== Closed
 
-    const drawerWidth =
+    const guide_drawer_selected =
```
Comment (FallingCeilingS) [inline src/browser/components/TabNavigation/Navigation.tsx:178]: Any reason why it's lower-kebab case?

**Review by FallingCeilingS — APPROVED** [review]
Nice improvement!
I only have a small question.

File: src/browser/components/TabNavigation/Navigation.tsx (hunk 1 above)
Comment (noahmay) [reply src/browser/components/TabNavigation/Navigation.tsx]: No, it's my mistake. Fixing!

**Comment by FallingCeilingS** [comment]
@noahmay - please feel free to merge once all checks turn green!

**Pull request merged**